Assets/_Project/Scripts/Level/LevelSelectionManager.cs

```diff
@@ -85,10 +85,6 @@ public override void DoStart()
         this.DestroyAndRefreshLevels();
     }
 
-    private void OnEnable() => SceneTransitionManager.OnSceneRemoved += OnSceneClosed;
-
-    private void OnDisable() => SceneTransitionManager.OnSceneRemoved -= OnSceneClosed;
-
     private void OnFilterLevels()
     {
```

Comment:
Lol, This didnt work anyway. Its a scriptable object so you would need to us DoOnEnable not OnEnable